packages/plugins/users-permissions/server/controllers/auth.js

```diff
@@ -100,6 +100,10 @@ module.exports = {
     try {
       const user = await getService('providers').connect(provider, ctx.query);
 
+      if (user.blocked === true) {
+        throw new ApplicationError('Your account is blocked!');
```

Comment:
I think this might be an `UnauthorizedError` instead an `ApplicationError`, wdyt?

Reply:
The user is providing auth, so I think it should be a Forbidden